.travis.yml

```diff
@@ -2,20 +2,26 @@ language: python
 matrix:
   include:
     - name: Plone 5.0
-      python: 2.7
+      python: 2.7.17
       env: PLONE_VERSION=50
     - name: Plone 5.1
-      python: 2.7
+      python: 2.7.17
       env: PLONE_VERSION=51
     - name: Plone 5.2/2.7
-      python: 2.7
+      python: 2.7.17
       env: PLONE_VERSION=52
     - name: Plone 5.2/3.6
       python: 3.6
       env: PLONE_VERSION=52
 
 before_script:
   - sleep 10
+
+before_install:
+  - sudo apt-get install -y graphicsmagick poppler-utils poppler-data
+  - sudo apt-get install -y ghostscript qpdf tesseract-ocr tesseract-ocr-eng
+  - sudo apt-get install -y libreoffice
+
 install:
   - mkdir -p buildout-cache/{eggs,downloads}
   - pip install -r requirements.txt
```

Comment:
I would also make the requirements conditional, so that we can drop the setuptools and zc.buildout pins from versions.cfg

Reply:
Without those requirements documentviewer would not be able to work at all.

Reply:
I think we have a misunderstanding here :) 
My comment wa about `requirements.txt` not the apt packages ;)
I am really happy this PR removes pdftk (I think it was also removed from ubuntu at a certain point in time), by the way.

About the requirements I thought about something like:
```
  - if [ "$PLONE_VERSION" == "51" ]; then pip install -r requirements-51.txt; fi
```
and so on.

Reply:
But wouldn't it make the requirements and the documentation unnecessarily complicated?

Reply:
I actually tried that myself and found an even better way (IMO): see https://github.com/collective/collective.documentviewer/pull/80/files
Would you mind to review it?